docs/releases/1.10.txt

```diff
@@ -322,7 +322,8 @@ Templates
 Tests
 ~~~~~
 
-* ...
+* Changed :class:`~django.test.TestCase` to check deferrable database
+  constraints at the end of tests.
```

Comment:
How does this sound for a rationale? Were you looking for something more detailed than this? Suggestions welcome.

```
* :class:`~django.test.TestCase` now checks deferrable database constraints at
  the end of each test. This could help catch application bugs earlier in the
  development process.
```